drivers/hmis/app/models/hmis/hud/household.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+class Hmis::Hud::Household < Hmis::Hud::Base
+  include ::Hmis::Concerns::HmisArelHelper
+
+  self.table_name = :hmis_households
+  self.primary_key = 'id'
+
```

Comment:
Pretty sure you want 
```
 def readonly?
  true
 end
```
in here somewhere

Reply:
I agree, we should add this so it errors if an update is attempted. Readonly apparently doesn't handle destroys which is why the `CREATE RULE attempt_hmis_households_del AS ON DELETE TO hmis_households DO INSTEAD NOTHING;` is necessary - is that right @ZachCMP ?

Reply:
That's my understanding based on my discussions with @ttoomey